[Ader] 카테고리 목록 조회 Mock API 구현 #7

ak2j38 opened the pull request:
- [x] CategoryResponse 구현
- [x] CategoryService 구현
- [x] CategoryController 구현
- [x] Swagger 기능 적용

closed Issue #7

ak2j38 committed:
feat: 카테고리 목록 조회 Mock API 구현

1. CategoryResponse 구현
2. CategoryService 구현
3. CategoryController 구현
4. Swagger 기능 적용

end

jinan159 commented on backend/src/main/java/com/team28/sidedish/controller/CategoryController.java:
```diff
@@ -0,0 +1,44 @@
+                        description = "카테고리 목록 조회 성공",
+                        content = {
+                            @Content(
+                                    mediaType = "application",
```
mediaType 은 `application/json` 이 되어야 할 것 같습니다!

ak2j38 replied:
앗.... 감사합니다! 어쩐지 스웨거 UI에서 확인해보니 `application`으로만 뜨더라구요! 
수정해서 PUSH 할게요!

jinan159 commented on backend/src/main/java/com/team28/sidedish/service/MockCategoryService.java:
```diff
@@ -0,0 +1,24 @@
+import java.util.List;
+
+@Service
+@Primary
```
현재는 `CategoryService` 타입의 빈이 `MockCategoryService` 하나밖에 없으니, 현재 단계에서는 `@Primary` 어노테이션이 없어도 될 것 같습니다!

ak2j38 replied:
금방 Jdbc 버전을 만들거라고 생각하고 넣어놨는데 현재는 필요없는 점이 맞습니다. 
수정할게요!

ak2j38 committed:
fix: 코드리뷰 수정사항 적용

1. swagger 설정 중 mediaType에 /json 누락된 부분 추가
2. 현재 service가 하나이므로 @Primary 어노테이션 제거

end

jinan159 commented:
벌써 반영 해주셨네요!
merge 하겠습니닷👍👍
